benchmarking/nursery/benchmark_multiobjective/README.md

```diff
@@ -0,0 +1,17 @@
+# Benchmark Multiobjective Methods in Syne Tune
+
+This folder shows one way to run quick experiments running different scheduler on different benchmarks and plot
+ results once they are done.
+
+To run all experiments, you can run the following:
+
+
+```bash
+pip install -r benchmarking/nursery/benchmark_multiobjective/requirements.txt
+python benchmarking/nursery/benchmark_multiobjective/benchmark_main.py --experiment_tag "my-new-experiment" --num_seeds 2
```

Comment:
Why not call this `hpo_main.py`?

And you could also add remote launching, so people don't need to worry about requirements, or running things locally.

Reply:
Renamed to `hpo_main.py`. Agree remote launching would be useful, can add in a future PR